### ISSUE TITLE
How to make gt for informative anchor proposal  branch and depth estimation branch?

### ISSUE THREAD

**Issue by sgzqc** [zhangboshen/A2J#3]
Hello!
You're work is very interesting but i have a question that how to make gt for informative anchor proposal branch and depth estimation branch?

**Comment by zhangboshen** [comment]
> Hello!
> You're work is very interesting but i have a question that how to make gt for informative anchor proposal branch and depth estimation branch?

Thanks for paying attention to our work.
There is no extra supervision needed for training informative anchor proposal branch, since we only have 2 loss functions, and only original annotation is needed (i.e., 3D joint coordinates). And 3D joint coordinates can also supervise depth estimation branch.

**Comment by sgzqc** [comment]
Thanks for your reply and the published code.However i only find the inference script , when will  the train script be available?

**Comment by zhangboshen** [comment]
> Thanks for your reply and the published code.However i only find the inference script , when will the train script be available?

Hi, I am sorry that currently we can only release inference code, due to some reasons. But you are very welcome to reproduce our training code by following the implementation details in our paper.
